### PR TITLE
Let python handle default arguments instead of PhySL.

### PR DESCRIPTION
**Pull request by stevenrbrandt**
Fixes issue #1206

### CODE REVIEW

File: tests/unit/python/execution_tree/eval.py [hunk 1]
```diff
@@ -270,3 +270,13 @@ def get_types():
     np.dtype('float64'),
     np.dtype('O'),
     np.dtype('O')], types
+
+default_val = 57
+
+
```
Comment (rtohid) [inline tests/unit/python/execution_tree/eval.py:276]: Could you please add the following tests too?

```py
@Phylanx
def usedefault(*args, arg=default_val):
    return arg


assert default_val == usedefault()


@Phylanx
def usedefault(arg=default_val):
    return arg


assert default_val == usedefault(arg=default_val))
```

File: python/phylanx/ast/transducer.py [hunk 2]
```diff
@@ -29,6 +29,19 @@ def lambda_counter(init=[0]):
     return init[0]
 
 
+def remove_defaults(tree):
```
[the remark this answers is not in the document]
Comment (stevenrbrandt) [reply python/phylanx/ast/transducer.py:32]: @hkaiser no, I didn't... I'll have to check.

[the remark this answers is not in the document]
Comment (stevenrbrandt) [reply python/phylanx/ast/transducer.py:32]: @rtohid I think it's the simplest way. The other would be to find where the defaults are read by the code generator and teach it to ignore them.

**Comment by hkaiser** [comment]
Closing this as #1288 has a proper fix.

**Pull request closed without merging**